internal/server/router/product_catalog.go

```diff
@@ -0,0 +1,48 @@
+package router
+
+import (
+	"net/http"
+
+	"github.com/openmeterio/openmeter/api"
+)
+
+// List features
+// (GET /api/v1/features)
+func (a *Router) ListFeatures(w http.ResponseWriter, r *http.Request, params api.ListFeaturesParams) {
+	if !a.config.EntitlementsEnabled {
```

Comment:
why not pass a NoopProductCatalogConnector in main instead?

Reply:
because we don't have a NoopProductCatalogConnector and i'm not convinced we need one, we should figure out conditional module loading instead IMO